common/http.go

```diff
@@ -0,0 +1,131 @@
+package common
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"gopkg.in/go-playground/validator.v9"
+
+	"github.com/pkg/errors"
+)
+
+type ErrorResponse struct {
+	Success bool `json:"success,omitempty"`
```

Comment:
Since `success` is always set to `false`, `omitempty` will make it disappear. 
Maybe we should remove it, but then from the code below as well.